<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusBarUI</class>
 <widget class="QWidget" name="StatusBarUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="frameNumLabel">
       <property name="text">
        <string>frame_num</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="frameFPSLabel">
       <property name="text">
        <string>frame_fps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="qNumLabel">
       <property name="text">
        <string>q_num</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sizeLabel">
       <property name="text">
        <string>proc_size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="frameTimeLabel">
       <property name="text">
        <string>frame_time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="processBitrateLabel">
       <property name="text">
        <string>proc_bitrate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="processSpeedLabel">
       <property name="text">
        <string>speed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="durationTimeLabel">
     <property name="text">
      <string>Duration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="statProgressBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
